keras_nlp/tokenizers/word_piece_tokenizer_test.py

```diff
@@ -1,245 +1,278 @@
-        dataset = dataset.map(tokenizer)
-        dataset = dataset.apply(
-            tf.data.experimental.dense_to_ragged_batch(batch_size=1)
-        )
-        element = dataset.take(1).get_single_element().numpy()
-        self.assertAllEqual(element, [[1, 2, 3]])
-
-    def test_from_file(self):
-        vocab_path = os.path.join(self.get_temp_dir(), "vocab.txt")
-        input_data = ["the quick brown fox."]
-        vocab_data = ["[UNK]", "the", "qu", "##ick", "br", "##own", "fox", "."]
-        with tf.io.gfile.GFile(vocab_path, "w") as file:
-            for piece in vocab_data:
-                file.write(piece + "\n")
-        tokenizer = WordPieceTokenizer(vocabulary=vocab_path)
-        call_output = tokenizer(input_data)
-        self.assertAllEqual(call_output, [[1, 2, 3, 4, 5, 6, 7]])
-
-    def test_config(self):
-        input_data = ["quick brOWN whale"]
-        vocab_data = ["@UNK@", "qu", "@@ick", "br", "@@OWN", "fox"]
-        original_tokenizer = WordPieceTokenizer(
-            vocabulary=vocab_data,
-            lowercase=False,
-            oov_token="@UNK@",
-            suffix_indicator="@@",
-            dtype="string",
-        )
-        cloned_tokenizer = WordPieceTokenizer.from_config(
-            original_tokenizer.get_config()
-        )
-        self.assertAllEqual(
-            original_tokenizer(input_data),
-            cloned_tokenizer(input_data),
-        )
-
-    def test_saving(self):
-        input_data = tf.constant(["quick brOWN whale"])
-        vocab_data = ["@UNK@", "qu", "@@ick", "br", "@@OWN", "fox"]
-        tokenizer = WordPieceTokenizer(
-            vocabulary=vocab_data,
-            lowercase=False,
-            oov_token="@UNK@",
-            suffix_indicator="@@",
-            dtype="string",
-        )
-        inputs = keras.Input(dtype="string", shape=())
-        outputs = tokenizer(inputs)
-        model = keras.Model(inputs, outputs)
-        model.save(self.get_temp_dir())
-        restored_model = keras.models.load_model(self.get_temp_dir())
-        self.assertAllEqual(
-            model(input_data),
-            restored_model(input_data),
-        )
-
-    def test_no_oov_token_in_vocabulary(self):
-        vocab_data = ["qu", "@@ick", "br", "@@OWN", "fox"]
-        with self.assertRaises(RuntimeError):
-            WordPieceTokenizer(
-                vocabulary=vocab_data,
-            )
-
-        vocab_data = ["@UNK@", "qu", "@@ick", "br", "@@OWN", "fox"]
-        with self.assertRaises(RuntimeError):
-            WordPieceTokenizer(
-                vocabulary=vocab_data,
-            )
-
-        vocab_data = ["UNK", "qu", "@@ick", "br", "@@OWN", "fox"]
-        with self.assertRaises(RuntimeError):
-            WordPieceTokenizer(
-                vocabulary=vocab_data,
-            )
-
-        with self.assertRaises(ValueError):
-            WordPieceTokenizer(vocabulary=vocab_data, oov_token=None)
+# Copyright 2022 The KerasNLP Authors

```

Comment:
Why is this entire file showing up as modified in the diff? Something weird is going on here.

Please only modify what is needed to test the added functionality.